src/pages/AddData.tsx

```diff
@@ -101,5 +100,4 @@ const AddDataLoader = () => {
 	);
 };
 
-const AddData = withOidcSecure(AddDataLoader);
-export { AddData };
+export { AddDataLoader as AddData };
```

Comment:
I had two choices:

1. Keep the external surface area of these files the same by rewriting the exports like this
2. Export the `Loader`-suffixed versions and change all the imports

I chose this lighter-touch version. @jasonaowen and I had a very long discussion [here](https://github.com/PhilanthropyDataCommons/front-end/pull/106#discussion_r1186069693) about the `Loader` naming convention, which neither of us _love_ but which we both decided wasn't worth belaboring at the time.

I will note that we now stepped straight into this concern of Jason's:

> One of the reasons we have the no-default-export rule enabled is to avoid that kind of acrobatics, because calling something one name in one place and another name in another place makes it more difficult to understand the code.

However, this part is no longer the case:

> And it's not exactly that it's insecure, it's that it's meant to be used inside a secured component.

Because we actually use some of these `Loader`s on public pages. And we we've been inconsistent with our use of `Loader`s anyway.

Really the whole component naming convention is due a rethink in light of our current complexity, but I think we should punt on that until Phase 3.